jcl/src/java.base/share/classes/com/ibm/oti/vm/VMLangAccess.java

```diff
@@ -122,4 +129,19 @@
 	public Package getSystemPackage(String name);
 	/*[ENDIF]*/
 	
+	/**
+	 * Returns a J9ClassRef object.
```

Comment:
I didn't notice earlier. This comment has the same problem fixed in VMAccess.java, referring to J9ClassRef and J9Class.